### PR TITLE
Create rake to persist procedures

### PR DESCRIPTION
**Pull request by philipeleandro**
Obs.: 
- The changes in `lib/data/procedures.csv` was made with @danielnottingham to remove invalid lines in csv file and fix two lines with correct code and port.
- Talking to @danielnottingham with agree to create a new script to persist cbhpm_procedures in the same process that procedures are created. That's why I needed to modify `Scripts::ReadProceduresPDF` to export a new attribute `anesthetic_port` in JSON file.

Steps to run rake:
1 - Run `rake port_values_2008:import`;
2 - Run `rake procedures:create_json_file["lib/data/procedures.csv",500]` to create JSON files with 500 lines from CSV file;
3 -  Run `rake procedures:persist_in_database["PATH JSON FILE"]` to persist procedures and cbhpm_procedures in database for each JSON file created in the last step.

### CODE REVIEW

**Review by danielnottingham — APPROVED** [review]

**Comment by danielnottingham** [comment]
LGTM! ❤️👏

suggestion: rebase/squash these commits

File: lib/scripts/persist_procedures.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    def save_procedure!(procedure_instance, port, anesthetic_port)
+      success_message = success_message(procedure_instance.code)
+
+      puts_message(success_message)
+
+      create_cbhpm_procedures(procedure_instance, port, anesthetic_port) if procedure_instance.save!
+    rescue StandardError => e
+      fail_message = fail_message(procedure_instance.code, e)
+
+      logger.warn(fail_message)
```
Comment (edimossilva) [inline lib/scripts/persist_procedures.rb:43]: maybe `log.error`?

File: lib/scripts/persist_procedures.rb [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      raise StandardError, e.message
```
Comment (edimossilva) [inline lib/scripts/persist_procedures.rb:44]: This sounds strange

Comment (philipeleandro) [reply lib/scripts/persist_procedures.rb:44]: I preferred to remove logger.warn and let only raise error. This current [PR](https://github.com/espoo-dev/med_system_backend/pull/183/commits/fab92b0a6a9c794ff1d20321c63b85f94fb1effa) was merged, so I created a new PR and this is the commit [link](https://github.com/espoo-dev/med_system_backend/pull/183/commits/fab92b0a6a9c794ff1d20321c63b85f94fb1effa)

File: lib/scripts/persist_procedures.rb (hunk 1 above)
Comment (philipeleandro) [reply lib/scripts/persist_procedures.rb:43]: I preferred to remove logger.warn and let only raise error. This current [PR](https://github.com/espoo-dev/med_system_backend/pull/183/commits/fab92b0a6a9c794ff1d20321c63b85f94fb1effa) was merged, so I created a new PR and this is the commit [link](https://github.com/espoo-dev/med_system_backend/pull/183/commits/fab92b0a6a9c794ff1d20321c63b85f94fb1effa)

**Pull request merged**